src/components/Col/_col.scss

```diff
@@ -0,0 +1,180 @@
+//-----------------------------
+// Grid.col
+//-----------------------------
+
+@import '../../globals/scss/colors';
+@import '../../globals/scss/vars';
+@import '../../globals/scss/layout';
+@import '../../globals/scss/typography';
+@import '../../globals/scss/helper-mixins';
+@import '@carbon/import-once/scss/import-once';
+@import '../../globals/scss/css--reset';
+
+@include exports('grid-col') {
+  .#{$prefix}--grid-col {
+    height: 100%;
+    box-sizing: border-box;
+    -webkit-box-flex: 0;
+  }
+  .#{$prefix}--col--span-2,
+  .#{$prefix}--col--lg--2 {
+    flex: 0 0 (1/6 * 100%);
```

Comment:
Could you write a mixin which calculates the different steps?

Reply:
mixin added